service_mesh/v2x/prepare-to-deploy-applications-ossm.adoc

```diff
@@ -14,6 +14,11 @@ ifdef::openshift-enterprise[]
 For more information, see the {product-title} documentation, xref:../../applications/deployments/what-deployments-are.adoc[Understanding deployments].
 endif::[]
 
+[NOTE]
+====
+Traffic started by `initContainer` cannot travel outside of the service mesh by default. Any action `initContainer` must perform that requires establishing a network traffic connection outside of the mesh will fail. For more information about connecting `initContainer` to a service, see the Red Hat Knowledgebase solution link:https://access.redhat.com/solutions/6963263[initContainer in CrashLoopBackOff when using Service Mesh and sending traffic to a host which requires DNS resolution].
```

Comment:
What is initContainer ? A CR, object, or something else? I think including a noun on the first instance would help.
-------------
The IBM SG states "Use past or future tense only when you cannot use present tense or it does not make sense to use present
tense.", so maybe convert "will fail" to "fails"?
---------
The "For more information..." could be placed on a new paragraph so the note does not look too dense. 
----------
Can we reference knowledge base articles in the customer documentation? What if the article is taken offline or drastically changed? How will we know?

Reply:
The title syntax of the link does match the article title, but the title is very long. The RH SGG suggests:

![image](https://github.com/openshift/openshift-docs/assets/57954076/f042e431-21bf-412c-a2da-d7cdf74a9545)

Reply:
If Kbase articles are OK to use, maybe the article number could be used instead of the long title.

Reply:
@dfitzmau thank you for reviewing this PR and the suggestions you've provided. I updated the text, and also changed will fail to fails.

We link to Kbase articles in other parts of our doc. Previously, I was coached that the link text should try to match the name or title of what you link to.

https://redhat-documentation.github.io/supplementary-style-guide/#accessibility-links-hypertext

I agree that the title is long. While I think using the article number would certainly be shorter, I also think it might introduce some ambiguity. My preference would be to have the longer link language so it matches the destination we're linking to.